src/SfxWeb/src/app/views/node/details/details.component.html

```diff
@@ -1,3 +1,26 @@
+<div class="detail-pane essen-pane">
```

Comment:
im still hesitant to move the health events to a separate page.

Reply:
We can discuss this further with Sridhar